fortnox-login.user.js

```diff
@@ -0,0 +1,16 @@
+// ==UserScript==
+// @name        Fortnox login
+// @version     1.0
+// @grant       none
+// @require     https://code.jquery.com/jquery-3.4.1.slim.min.js
+// @include     https://*.fortnox.se/*
+// @downloadURL https://github.com/Wikimedia-Sverige/scripts/raw/master/fortnox-login.user.js
+// ==/UserScript==
+
+$(function() {
+  var anchors = document.querySelectorAll('a.Header_loginBtn__wQ_VG');
+    for (var i = 0; i < anchors.length; i++) {
```

Comment:
Can be replaced with
`for(var anchor of anchors)`
or the whole loop with
`anchors.forEach(e => e.setAttribute(...))`
to make it more compact.

Reply:
Also, are there ever more than one login button?

Reply:
> Also, are there ever more than one login button?

I honestly don't remember. Likely not ?

Reply:
> Can be replaced with `for(var anchor of anchors)` or the whole loop with `anchors.forEach(e => e.setAttribute(...))` to make it more compact.

Thanks. Implemented